jbpm-system-test/jbpm-unmarshall-ejb-timers-test/src/test/java/org/kie/samples/integration/UnmarshallEJBTimerSystemTest.java

```diff
@@ -0,0 +1,220 @@
+                                        .withUsername("rhpamuser")
+                                        .withPassword("rhpampassword")
+                                        .withFileSystemBind("target/postgresql", "/docker-entrypoint-initdb.d",
+                                                            BindMode.READ_ONLY)
+                                        .withNetwork(network)
+                                        .withNetworkAliases("postgresql11");
+
+    @ClassRule
+    public static KieServerContainer kieServer1 = new KieServerContainer("node1", network, args);
+    
+    @ClassRule
+    public static KieServerContainer kieServer2 = new KieServerContainer("node2", network, args);
+    
+    private static KieServicesClient ksClient1;
+    private static KieServicesClient ksClient2;
+    
+    private static ProcessServicesClient processClient1;
+    private static ProcessServicesClient processClient2;
+    
+    private static HikariDataSource ds;
+    
+    @BeforeClass
+    public static void setup() {
+        logger.info("KIE SERVER 1 started at port "+kieServer1.getKiePort());
+        logger.info("KIE SERVER 2 started at port "+kieServer2.getKiePort());
+        logger.info("postgresql started at "+postgreSQLContainer.getJdbcUrl());
+        
+        ksClient1 = authenticate(kieServer1.getKiePort(), DEFAULT_USER, DEFAULT_PASSWORD);
+        ksClient2 = authenticate(kieServer2.getKiePort(), DEFAULT_USER, DEFAULT_PASSWORD);
+        
+        processClient1 = ksClient1.getServicesClient(ProcessServicesClient.class);
+        processClient2 = ksClient2.getServicesClient(ProcessServicesClient.class);
+        
+        createContainer(ksClient1);
+        createContainer(ksClient2);
+        
+        ds = getDataSource();
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        ksClient1.disposeContainer(containerId);
+        ksClient2.disposeContainer(containerId);
+        DockerClient docker = DockerClientFactory.instance().client();
+        docker.listImagesCmd().withLabelFilter("autodelete=true").exec().stream()
+         .filter(c -> c.getId() != null)
+         .forEach(c -> docker.removeImageCmd(c.getId()).withForce(true).exec());
+    }
+
+    @Test
+    @DisplayName("user starts process in one node but get variable in another before refresh-time")
+    public void getVariableBeforeRefresh() throws Exception {
+        Long processInstanceId = startProcessInNode1("ht-timer-process");
+        
+        Thread.sleep(2000);
+        
+        findVariableInNode2(processInstanceId);
+        
+        assertEquals("there should be just one timer at the table",
+                      1, performQuery(SELECT_COUNT_FROM_JBOSS_EJB_TIMER).getInt(1));
+        
+        assertEquals("timer should be started at node1 partition",
+                     "ejb_timer_node1_part", performQuery(SELECT_PARTITION_NAME_FROM_JBOSS_EJB_TIMER).getString(1));
```

Comment:
this assert will fail when the `org.kie.samples.ejbtimer.nocluster` is enabled as per https://github.com/kiegroup/jbpm-playground/blob/6b9de0deac8f4088daa61d17a0eee46e298b20de/jbpm-system-test/jbpm-unmarshall-ejb-timers-test/src/test/java/org/kie/samples/integration/UnmarshallEJBTimerSystemTest.java#L166-L168

Maybe we can get rid of that property since it doesn't make sense in this test

Reply:
keep it for allowing that scenario without cluster